### PR TITLE
Tweak CSS of overlay icon and progress bar

### LINKED ISSUE #128: CSS tweak for overlay icon?
- on hover, top corners aren't rounded and bottom background dimensions are off

**vimeo, large vid on homepage**

screen shot 2016-08-17 at 10 51 09 am

**youtube, small vid on homepage**
screen shot 2016-08-19 at 1 23 01 pm

### PR DESCRIPTION
**Pull request by kaganjd**
Fixes #128.

### CODE REVIEW

File: data/panel.css [hunk 1]
```diff
@@ -229,7 +230,6 @@ input[type=range]::-moz-focus-outer {
 
 .video-progress {
     background: #ffffffaa;
-    border-radius: 20px;
```
Comment (meandavejustice) [inline data/panel.css]: why did you remove border-radius here and below in `.video-progress[value]::-moz-progress-bar`?

Comment (kaganjd) [inline data/panel.css]: @6a68 went back and forth about different ways to get the `[value]` to be the same shape as the `.video-progress` bar at < 2 seconds. I looked at svg masks, the background-clip css property, and few other things, but eventually it seemed to be a lot of work for such a small thing. I also wasn't sure how a mask would scale as the player gets resized. So we decided that the way to most consistently get the [value] bar to be the right shape was just to get rid of the border-radius.

Comment (kaganjd) [inline data/panel.css]: Here's what it looks like with padding-left. A little better, but still not quite right.
screen shot 2016-08-29 at 4 47 28 pm

**Comment by kaganjd** [comment]
This PR also doesn't address the `overlay-icon` interfering with some Vimeo and YT elements because changing the location of the icon on the page seems like something that warrants more UI discussion with @johngruen. I'll create a separate issue for that with the screenshots from #128

**Comment by meandavejustice** [comment]
The border radius changes here seem like a regression to me. @kaganjd Would you mind discussing this further? Either here on github or in irc
